src/components/SidePanels/OutlinerPanel/OutlinerTree.tsx

```diff
@@ -129,13 +129,33 @@ const useDataTree = ({
     const style = item?.background ?? item?.getOneColor?.();
     const color = chroma(style ?? '#666').alpha(1);
     const mods: Record<string, any> = { hidden, type, isDrawing };
+
     const label = (() => {
       if (!type) {
         return 'No Label';
       } else if (type.includes('label')) {
         return item.value;
       } else if (type.includes('region') || type.includes('range')) {
-        return (item?.labels ?? []).join(', ') || 'No label';
+        const labelsInResults = item.results
+          .filter((result: any) => result.type.endsWith('labels'))
```

Comment:
you can type array itself so next array methods don't need to be typed: `(item.results as any)`

Reply:
also you can use `item.labelings` which return results with labels

Reply:
done